### ISSUE TITLE
k3s crictl --version 

### ISSUE THREAD

**Issue by lygstate** [rancher/k3s#236]
The crictl --verion result is unknow
```
root@GBPSCLOUD-190128-TXC4:~# k3s crictl --version
crictl github.com/containerd/containerd unknown
```

**Comment by warmchang** [comment]
This is an bug from cri-tools, and has been fixed in v1.11.1.

FYI: https://github.com/kubernetes-sigs/cri-tools/issues/353#issuecomment-460671416

Maybe we need upgrade the repo tag in "vendor.conf" for the next release.

BTW, the "k3s crictl version" runs well:

> $ k3s crictl version
> Version:  0.1.0
> RuntimeName:  containerd
> RuntimeVersion:  1.2.4+unknown
> RuntimeApiVersion:  v1alpha2
> $

**Comment by erikwilson** [comment]
This should be fixed with #626 and available for testing in https://github.com/rancher/k3s/releases/tag/v0.7.0-rc5. Thanks for reporting @lygstate!

**Comment by galal-hussein** [comment]
k3s version: v0.7.0-rc5

I can see k8s version appear correctly:
```
k3s crictl --version
crictl github.com/rancher/containerd v1.14.0-k3s1 
```
